notebooks/233-encodec-audio-compression/233-encodec-audio-compression.ipynb

```diff
@@ -4,7 +4,26 @@
    "cell_type": "markdown",
    "id": "285a34a4-18ff-493c-8322-71adf1f09622",
    "metadata": {},
-   "source": []
+   "source": [
+    "# Audio compression with EnCodec and OpenVINO\n",
+    "\n",
+    "Compression is an important part of the internet today, because it enables people to easily share high-quality photos, listen to audio messages, stream their favorite shows, and so much more. Even when using today’s state-of-the-art techniques, enjoying these rich multimedia experiences requires a speedy internet connection and plenty of storage space. AI helps to overcome these limitations: \"Imagine listening to a friend’s audio message in an area with low connectivity and not having it stall or glitch.\" \n",
+    "\n",
+    "In this tutorial, we consider how to use  OpenVINO and EnCodec algorithm for hypercompression of audio.\n",
+    "EnCodec is a real-time, high-fidelity audio codec that uses AI to compress audio files without losing quality. It was introduced in [High Fidelity Neural Audio Compression](https://arxiv.org/pdf/2210.13438.pdf) paper by Meta AI. The researchers claim they achieved an approximate 10x compression rate without a loss of quality and made it work for CD-quality audio. More details about this approach can be found in [Meta AI blog](https://ai.facebook.com/blog/ai-powered-audio-compression-technique/) and original [repo](https://github.com/facebookresearch/encodec)\n",
```

Comment:
Suggested:
EnCodec is a real-time, high-fidelity audio codec that uses AI to compress audio files without losing quality. It was introduced in [High Fidelity Neural Audio Compression](https://arxiv.org/pdf/2210.13438.pdf) paper by Meta AI. The researchers claimed they achieved an approximate 10x compression rate without loss of quality and made it work for CD-quality audio. More details about this approach can be found in [Meta AI blog](https://ai.facebook.com/blog/ai-powered-audio-compression-technique/) and original [repo](https://github.com/facebookresearch/encodec). \n